Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -999,17 +1002,29 @@ void MenuBar::ImportWiiSave()
 
   bool cancelled = false;
   auto can_overwrite = [&] {
-    bool yes = QMessageBox::question(
-                   this, tr("Save Import"),
-                   tr("Save data for this title already exists in the NAND. Consider backing up "
-                      "the current data before overwriting.\nOverwrite now?")) == QMessageBox::Yes;
+    QMessageBox msg(QMessageBox::Warning, tr("Save Import"),
+                    tr("Save data for this title already exists in the NAND. Consider backing up "
+                       "the current data before overwriting.\nOverwrite now?"),
+                    QMessageBox::Yes | QMessageBox::No, this);
+    msg.setWindowModality(Qt::WindowModal);
+    bool yes = msg.exec() == QMessageBox::Yes;
     cancelled = !yes;
     return yes;
   };
+
+  QMessageBox msg(this);
+  msg.setWindowModality(Qt::WindowModal);
+  msg.setWindowTitle(tr("Save Import"));
+
   if (WiiSave::Import(file.toStdString(), can_overwrite))
-    QMessageBox::information(this, tr("Save Import"), tr("Successfully imported save files."));
+  {
+    msg.setText(tr("Successfully imported save files."));
+  }
   else if (!cancelled)
-    QMessageBox::critical(this, tr("Save Import"), tr("Failed to import save files."));
+  {
+    msg.setText(tr("Failed to import save files."));
```

Comment:
This one (and the one above) have no explicit type, so at least one of them will have a wrong icon.